### PR TITLE
Create labelColor prop within AbstractChart class for separate label/chart colors

### PR DESCRIPTION
**Pull request by slavkobojanic**
The .gitignore isn't necessary but didn't want my editor config files to clutter your repository. I've also gone ahead and slightly darkened the opacity of the label text because it was a bit too light (in my opinion, but can be changed back, your choice!).

The PieChart component has not been altered since the dataset has it's own legendFontColor/Size props so it'd be redundant there, but everywhere else has been updated.

I've also updated the readme to reflect the new labelColor prop which is used in exactly the same way as the already existing `color` prop.

I've tested them with the example app in the repo and everything *looks* fine to me. Let me know if anything needs improvement/changes :)

### CODE REVIEW

File: src/abstract-chart.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ class AbstractChart extends Component {
           textAnchor="end"
           y={(height * 3) / 4 - ((height - paddingTop) / count) * i + 12}
           fontSize={12}
-          fill={this.props.chartConfig.color(0.5)}
+          fill={this.props.chartConfig.labelColor(0.8) || '#ffffff'}
```
[the remark this answers is not in the document]
Comment (slavkobojanic) [reply src/abstract-chart.js]: Yeah no sweat, i'll refactor that as soon as I get a chance :)

**Comment by Hermanya** [comment]
I fixed a tiny merge conflict here

**Comment by rishab1065** [comment]
When this PR will be merged and available for use? I have the same requirement.

**Comment by slavkobojanic** [comment]
> When this PR will be merged and available for use? I have the same requirement.

I've been a bit hectic with a bunch of uni stuff at the moment, hopefully i'll be able to fix up the merge request soon. You're welcome to use my fork of it for the time being until I find a second to sort it out.

**Comment by Hermanya** [comment]
I gotcha @slavkobojanic, this is now available in v3.9.0

**Comment by slavkobojanic** [comment]
Thanks very much @Hermanya, much appreciated!!!

**Pull request merged**